agent/sshd/cmd.go

```diff
@@ -37,6 +34,6 @@ func newCmd(u *osauth.User, shell, term, host string, command ...string) *exec.C
 	}
 	cmd.Dir = u.HomeDir
 	cmd.SysProcAttr = &syscall.SysProcAttr{}
-	cmd.SysProcAttr.Credential = &syscall.Credential{Uid: uint32(uid), Gid: uint32(gid), Groups: groups}
+	cmd.SysProcAttr.Credential = &syscall.Credential{Uid: uint32(u.UID), Gid: uint32(u.GID), Groups: groups}
```

Comment:
Couldn't this use `u.UID` and `u.GID` directly?

Reply:
Yes it is, because the fields in the user struct use ints. I thought of changing that but it requieres casts in other places back to int as the os package wants ints, whereas the syscall package wants uint32

Either way there will be casts, let me know if you consider the other way around (casting when parsing) cleaner

Reply:
If the syscall requires it to be an `uint32` we could do the cast where we gather the information. So we do it once and avoid redoing it all over the place.